<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsAppareilMode</class>
 <widget class="QDialog" name="DetailsAppareilMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Details pour Chauffage Salon</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Vendredi</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>Samedi</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string>Dimanche</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Jeudi</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Mercredi</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>Mardi</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>Lundi</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>401</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>Tous les jours</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Toutes les semaines</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Nouveau créneau</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Modifier créneau</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Supprimer créneau</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <attribute name="headerDefaultSectionSize">
    <number>134</number>
   </attribute>
   <column>
    <property name="text">
     <string>Allumer à</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Eteindre à</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Thermostat</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DetailsAppareilMode</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DetailsAppareilMode</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
